examples/v1alpha3/nas/enas-cnn-cifar10/Dockerfile.gpu

```diff
@@ -2,6 +2,8 @@ ARG cuda_version=10.0
 ARG cudnn_version=7
```

Comment:
Please, also change NAS examples for v1beta1: https://github.com/kubeflow/katib/tree/master/examples/v1beta1/nas

Reply:
CHanged